src/much-select.js

```diff
@@ -1735,12 +1735,13 @@ class MuchSelect extends HTMLElement {
         In multi select mode, let's let the selected values line up on a row and then wrap.
         */
         flex-flow: row wrap;
+        margin-top:5px;
       }
 
       #value-casing.output-style-custom-html.multi .value {
-        border: 3px solid black;
-        min-width: 10px;
-
+        border: 2px solid black;
```

Comment:
(I know you didn't do this Ranjith but this color, in fact probably this whole border definition should be defined in a variable.